.github/workflows/e2e-tests.yml

```diff
@@ -0,0 +1,27 @@
+name: Test solidity contracts
+on:
+  push:
+  schedule:
+    - cron: '0 20 * * */1'
+jobs:
+  test-contracts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
+      - name: Install Dependencies
+        run: yarn
+      - name: Create .env
+        run: |
```

Comment:
Can we remove this if GitHub is able to obtain the secrets from `${{ secrets.ETHERSCAN_API_KEY }}` etc. Is there a reason for GitHub to also create a .env file?